test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -261,6 +270,8 @@ private static void validateSpec(RestSpec restSpec) {
 
     @After
     public void wipeCluster() throws Exception {
+        // Running tasks might create indices and otherwise invalidate the wipe. Lets just let the finish.
+        waitForAllRunningTasksToFinish();
```

Comment:
What do you mean by a "section", something in the rest test language? If so then I'm in favor of that. But then we should fail hard in rest test cleanup if there are any running tasks (ie the test isn't correctly waiting like it should).